dcos/dcos-ui
# [COPS] DCOS_OSS-4633: [1.13] dcos6 network is shown as available for UCR in the UI.

Make the virtual network dcos6 unavailable
when UCR is chosen as a container.

Closes https://jira.mesosphere.com/browse/DCOS_OSS-4633

## Testing
1. Go to Services tab
2. Create a new Single Container service
3. On the Service tab, click "More Settings" at the bottom and select "Universal Container Runtime (UCR)" for Container Runtime.
4. Go to Networking type, open the Network Type dropdown and verify that "dcos6" is not shown.

## Trade-offs
Somewhat quick and dirty solution. I would be fine if we decide that we don't want it and if this doesn't fix it, maybe it can at least point us in the right direction.

Edit: Also, should the same thing be done for pods?

## Dependencies
None.

## Screenshots
![peek 2019-01-04 14-14](https://user-images.githubusercontent.com/40791275/50687976-12b57c80-102c-11e9-99fd-ffdc1d2199c8.gif)
